app.js

```diff
@@ -1,130 +1,178 @@
-    let setArray = []
-    let itemOne = Math.floor(Math.random() * validProducts.length);
-    setArray.push(itemOne.image);
-    let itemTwo = Math.floor(Math.random() * validProducts.length);
-    setArray.push(itemTwo.image);
-    let itemThree = Math.floor(Math.random() * validProducts.length);
-    setArray.push(itemThree.image);
-    while(itemOne === itemTwo) {
-      itemTwo = Math.floor(Math.random() * validProducts.length);
-  }
-    while(itemOne === number3 || itemTwo === itemThree) {
-      itemThree = Math.floor(Math.random() * validProducts.length);
-  }
-    return setArray;
+let totalSets;
+    // this is a variable who value will be incremented with each display rendering
+
+const initiateState = () => {
+    productSelectionCount = [];
+    totalViews = [];
+    totalSets = 0;
 };
 
+initiateState();
 
 
 
-// render set array on webpage
-export const renderSetFunction = (newSetFunction) => {
-   let setArray[0] = item1;
-   let setArray[1] = item2;
-   let setArray[2] = item3;
+/// ---> SECOND : What will the user do to interact and modify these variables and update the state? AND, what does the site require for such an intervention?
+// 1. Display content
+    // need a number randomizer function that will generate nums that we can use as indicies to identify and return sets of items at random
+      // must return 3 dissimlar products 
+      
+  
 
-   photoOne.src = item1;
-   photoTwo.src = item2;
-   photoThree.src = item3;
+    //randomizer
+const randomProduct = (productsList) => {
+    const index = Math.floor(Math.random() * productsList.length);
+    const randomProduct = productsList[index];
+    return randomProduct;
+}
 
+  //function (which will be called in eventual eventListener) will first, utilize our randomizer to return 3 dissimlar products 
+const returnThreeProducts = () => {
+  
+    let productOne = randomProduct(productsList);
+    let productTwo = randomProduct(productsList);
+    while (productOne.id === productTwo.id) {
+        productTwo = randomProduct(productsList)
+    }
+    let productThree = randomProduct(productsList);
+    while (productThree === productTwo || productThree === productOne) {
+        productThree = randomProduct(productsList);
+    }
+
+/// 2: equipped with those products, now manipulate the         DOM, rendering the content
+        //create seperate variables for each State element and assign corresponding properties from generated products
+          // each property (image ; name ; id) will be assigned to a variable.
+              //what do id's link to?
+              //where do img's go? What is needed (src)?
+              //where will name appear? How (p, h)?
+
+   /// these are all NEW variables, grabbing DOM
+    // id's
+    const radioOneButton = document.getElementById('radio-one');
+    const radioTwoButton = document.getElementById('radio-two');
+    const radioThreeButton = document.getElementById('radio-three') ;
```

Comment:
Be careful, it looks like these strings not match the ids you have in your HTML.